docs/src/test/scala/docs/http/scaladsl/server/directives/CustomHttpMethodSpec.scala

```diff
@@ -30,7 +30,7 @@ class CustomHttpMethodSpec extends AkkaSpec with ScalaFutures
       //#application-custom
       import akka.http.scaladsl.settings.{ ParserSettings, ServerSettings }
 
-      // define custom media type:
+      // define custom method type:
       val BOLT = HttpMethod.custom("BOLT", safe = false,
         idempotent = true, requestEntityAcceptance = Expected)
 
```

Comment:
Move the ending `//#application-custom` 4 lines below (after `val response = Http().singleRequest(request).futureValue`), so the request example is included.